docs/integration/KafkaIntegration.md

```diff
@@ -44,4 +44,136 @@ In the Designer the user can choose, for each source and sink, which schema vers
 At runtime Nussknacker determines the schema version of a message value and key in the following way:
 1. it checks `value.schemaId` and `key.schemaId` headers;
 2. if there are no such headers, it looks for a magic byte and schema version in the message, [in a format used by Confluent](https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#wire-format);
-3. if the magic byte is not found, it assumes the schema version chosen by the user in the scenario.
\ No newline at end of file
+3. if the magic byte is not found, it assumes the schema version chosen by the user in the scenario.
+
+  
+# Configuration
+
+## Common part
```

Comment:
The key relative which Kafka is configured in Streaming-Flink and Streaming-Lite is (most likely) different. I think it should be mentioned here ..

Reply:
Added mention